kitchen-tests/.kitchen.travis.yml

```diff
@@ -1,35 +1,37 @@
 ---
-driver_config:
+driver:
+  name: ec2
   aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
   aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
   aws_ssh_key_id: <%= ENV['AWS_KEYPAIR_NAME'] %>
   iam_profile_name: <%= ENV['IAM_PROFILE_NAME'] %>
+  region: "us-west-2"
+  availability_zone: "us-west-2a"
+  security_group_ids: ["travis-ci"]
+
 
 provisioner:
   name: chef_zero
-  github: <%= ENV['TRAVIS_REPO_SLUG'] %>
-  branch: <%= ENV['TRAVIS_COMMIT'] %>
-  require_chef_omnibus: 12.0.0-rc.2
+  github_owner: "chef"
+  github_repo: "chef"
+  refname: <%= ENV['TRAVIS_COMMIT'] %>
   data_path: test/fixtures
 # disable file provider diffs so we don't overflow travis' line limit
   client_rb:
     diff_disabled: true
 
+transport:
+  ssh_key: <%= ENV['EC2_SSH_KEY_PATH'] %>
+
 platforms:
   - name: ubuntu-12.04
-    driver_plugin: ec2
-    driver_config:
-      region: "us-west-2"
-      availability_zone: "us-west-2a"
-      ssh_key: <%= ENV['EC2_SSH_KEY_PATH'] %>
-      security_group_ids: ["travis-ci"]
+    driver:
+      # http://cloud-images.ubuntu.com/locator/ec2/
+      image_id: ami-e1557fd1
   - name: centos-6.4
-    driver_plugin: ec2
-    driver_config:
-      region: "us-west-2"
-      availability_zone: "us-west-2a"
-      ssh_key: <%= ENV['EC2_SSH_KEY_PATH'] %>
-      security_group_ids: ["travis-ci"]
+    driver:
+      # https://aws.amazon.com/marketplace/seller-profile/ref=srh_res_product_vendor?id=16cb8b03-256e-4dde-8f34-1b0f377efe89
+      image_id: ami-b3bf2f83
```

Comment:
My memory on this was vague, but I remembered recently getting an error when trying to use an m1.small for something random and seeing the instance types had changes. Something's changing related to instance storage over at AWS, but I don't see any clear statements about it. The "m1" instances are being replaced by "t2" instances which don't support instance storage per [this page](http://aws.amazon.com/ec2/previous-generation/). Other instances still have instance storage but they're calling it "SSD" or "HDD" now on [the pricing page](http://aws.amazon.com/ec2/pricing/.

It's always seemed to me the value of instance storage over ebs storage is that you pay extra for how much ebs storage you used, but given that you don't have a choice in the small instances anymore, I guess you have to compare the cost of a t1 to an m3+ebs?